Diffusion/Common/ModelInfo.swift

```diff
@@ -202,6 +208,22 @@ extension ModelInfo {
         isXL: true
     )
 
+    static let sd3 = ModelInfo(
+        modelId: "argmaxinc/coreml-stable-diffusion-3-medium",
+        modelVersion: "SD3 medium (512, macOS)",
+        supportsEncoder: false,
+        quantized: false,
+        isSD3: true
+    )
```

Comment:
This model does not have a split einsum package; however, [some Macs](https://github.com/huggingface/swift-coreml-diffusers/blob/17ee716c8e2ac006a48403358181d6ab24302133/Diffusion/Common/ModelInfo.swift#L77) default to using the ANE with split einsum attention. What do you think we could do to override compute unit selection in this case, and force it to always be GPU?

Reply:
I took the inverse approach and only enabled it for .all and .gpu https://github.com/huggingface/swift-coreml-diffusers/pull/96/files#diff-eb8807a52606483b97fb37ae59fb50afcd9e0504aa9381d40727807badf31233R111. If user selects ANE it will show a pipeline error because the zip wont be found. We could make that a little cleaner though with a descriptive error, if you'd prefer.

Reply:
A better error sounds good if we can. I think it's ok to stop there unless the first-time launch for new users defaults to split einsum + sd3 on some Macs.

Reply:
I'm adding this to the selector so that folks will know when it is available or not. This way it will not fail and delete the model cache, since it will never attempt to download it in the first place.

Not supported:
<img width="294" alt="image" src="https://github.com/huggingface/swift-coreml-diffusers/assets/1981179/db005429-4bfe-431f-a2f6-f6ce59adbf64">
Supported:
<img width="293" alt="image" src="https://github.com/huggingface/swift-coreml-diffusers/assets/1981179/08be38b0-f6b2-4b07-8b92-21aaab8f3dd4">

It also will auto switch back to GPU if selecting a model where the previous compute unit was ANE.

Reply:
Is there a way to convert the SDXL and SD3 model into Split_Einsum? I have a MBP M2 and an iPad Pro M4 both with 16GB Ram and on both the ANE is a lot faster than CPU/GPU! For now I am using the 768x768 Split_Einsum SDXL model for iOS and I am wondering why there are no Split_Einsum versions of  those bigger models?